bulk-import/ndjson-generator/NDJsonGenerator.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace NDJsonGenerator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class NDJsonGenerator
+    {
+        private const string HeaderSection = "{\"Section\": \"Header\"}";
+        private const string ModelsSection = "{\"Section\": \"Models\"}";
+        private const string TwinsSection = "{\"Section\": \"Twins\"}";
+        private const string RelationshipsSection = "{\"Section\": \"Relationships\"}";
+        private const string DefaultHeader = "{\"fileVersion\": \"1.0.0\", \"author\": \"authorName\", \"organization\": \"organization\"}";
+
+        public static Stream GenerateNDJson(IEnumerable<string> models, IEnumerable<BasicDigitalTwin> twins, IEnumerable<BasicRelationship> relationships, string header = "")
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream, Encoding.UTF8);
+
+            writer.WriteLine(HeaderSection);
+            if (String.IsNullOrEmpty(header)) writer.WriteLine(DefaultHeader);
+            else writer.WriteLine(header);
+
+            if (models != null)
```

Comment:
To avoid the nesting at multiple levels, consider using:

```
models ??= Enumerable.Empty<string>();
twins ??= Enumerable.Empty<BasicDigitalTwin>();
...
```

That will assign all variables an empty list if they are null, and otherwise leave them in tact.